fastify.js

```diff
@@ -422,6 +427,10 @@ function build (options) {
     const reply = new Reply(req, res, null)
     reply.code(404).send(new Error('Not found'))
   }
+
+  function validatorMaker (schema) {
+    return ajv.compile(schema)
+  }
```

Comment:
can you just call it `validator`?

Reply:
But it is not a validator. validatorWrapper?  validatorBuilder?

Reply:
I like short names. How about `schemaBuilder`? `validatorFactory`? Naming things is hard.

cc @davidmarkclements

Reply:
`schemaCompiler`

Reply:
`schemaCompiler` is nice.

Reply:
> `schemaCompiler`

very nice! thanks @davidmarkclements !